ui/components/app/transaction-list-item-details/transaction-list-item-details.component.js

```diff
@@ -4,16 +4,17 @@ import copyToClipboard from 'copy-to-clipboard';
 import { getBlockExplorerLink } from '@metamask/etherscan-link';
```

Comment:
The changes in this file are responsible for the layout changes in the transaction details modal:

![Screenshot from 2021-12-01 09-45-09](https://user-images.githubusercontent.com/7499938/144241045-6ce9e14f-9608-4eaf-9a4a-fc75b76f7051.png)